src/Util/TimeFormater.js

```diff
@@ -0,0 +1,27 @@
+export default function timeSince(date) {
```

Comment:
do we want to write some basic unit tests for this?

Reply:
Added stackoverflow reference.